shell/browser/api/electron_api_safe_storage.cc

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2021 Slack Technologies, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "shell/browser/api/electron_api_safe_storage.h"
+
+#include "components/os_crypt/os_crypt.h"
+#include "shell/browser/browser.h"
+#include "shell/common/gin_converters/base_converter.h"
+#include "shell/common/gin_converters/callback_converter.h"
+#include "shell/common/gin_helper/dictionary.h"
+#include "shell/common/gin_helper/object_template_builder.h"
+#include "shell/common/node_includes.h"
+#include "shell/common/platform_util.h"
+
+namespace electron {
+
+namespace api {
+
+bool SafeStorage::electron_crypto_ready = false;
+gin::WrapperInfo SafeStorage::kWrapperInfo = {gin::kEmbedderNativeGin};
+
+electron::api::SafeStorage::SafeStorage(v8::Isolate* isolate) {}
```

Comment:
Looks like we don't need this `isolate`? Can probably drop it from the signature